Type error if compiled into a project with "skipLibCheck": false

**Describe the bug**

When compiling into a Typescript project you will get this error if you have "skipLibCheck" set to false.  This is usually an indicator that there is a problem with typing:

```
../../node_modules/@gltf-transform/functions/dist/resample.d.ts(2,31): error TS2307: Cannot find module 'keyframe-resample' or its corresponding type declarations.
```


**To Reproduce**

Compile the Node.js example code on the home page and be sure to reference @gltf-transform/functions

https://gltf-transform.dev

**Expected behavior**

No type errors emitted.

**Versions:**
 - Version: 3.4.2
 - Environment: Node.js 19.7.0
 - Typescript: 5.1.3

--- a/packages/functions/src/resample.ts
+++ b/packages/functions/src/resample.ts
@@ -21,7 +21,7 @@ const EMPTY_ARRAY = new Float32Array(0);
 
 export interface ResampleOptions {
 	ready?: Promise<void>;
-	resample?: typeof resampleDebug;
+	resample?: unknown; // glTF-Transform/issues/996
 	tolerance?: number;
 }
 
@@ -61,7 +61,7 @@ export function resample(_options: ResampleOptions = RESAMPLE_DEFAULTS): Transfo
 		const logger = document.getLogger();
 
 		const ready = options.ready;
-		const resample = options.resample;
+		const resample = options.resample as typeof resampleDebug;
 
 		await ready;
 

